pandas/plotting/_core.py

```diff
@@ -308,7 +308,7 @@ def hist_frame(
 
 See Also
 --------
-Series.plot.hist: Make a histogram.
+pandas.Series.plot.hist: Make a histogram.
```

Comment:
This renders properly? https://pandas.pydata.org/docs/dev/reference/api/pandas.DataFrame.boxplot.html

Reply:
Indeed this is rendered correctly now. It seemed to be broken yesterday. I will remove this change and resubmit the pull request.

Reply:
Ah nice catch, thx